schedule/__init__.py

```diff
@@ -246,6 +255,10 @@ def sunday(self):
         self.start_day = 'sunday'
         return self.weeks
 
+    def tag(self, *tags):
+        self.tags += tags
+        return self
```

Comment:
I had a thought about this before writing. What I thought was that a tag should actually be anything, also a tuple of tags so I choose to not enforce any kind of tag here.